bin/phpcs-autofix.sh

```diff
@@ -11,7 +11,5 @@
 default_path="."
 file_path=${1:-$default_path}
 
-ignore="\.idea,\.saas-cache,node_modules,vendor,bin,tests"
-
 echo Running PHPCS autofix
-./vendor/bin/phpcbf -v -d memory_limit="256M" --extensions="php" --ignore="${ignore}" ${file_path}
+./vendor/bin/phpcbf -v ${file_path}
```

Comment:
With the `-v` gone, this bash file becomes redundant, in so much as it provides no additional args to `phpcs`, than just calling `phpcs` directly. The only extra thing would be that `echo`. If that's not deemed critical, could the whole be removed (I'm not sure where it is used)?

Reply:
I like having the script so I (and others) don't have to always type the path to phpcbf etc

Reply:
Fair enough, but adding `./vendor/bin` to your `$PATH` (before ~/.composer/vendor/bin)` and this becomes un-necessary.